presto-native-execution/presto_cpp/main/CMakeLists.txt

```diff
@@ -108,3 +108,9 @@ set_property(TARGET presto_server PROPERTY JOB_POOL_LINK presto_link_job_pool)
 if(PRESTO_ENABLE_TESTING)
   add_subdirectory(tests)
 endif()
+
+if(PRESTO_ENABLE_PROMETHEUS_REPORTER)
+  find_package(prometheus-cpp CONFIG REQUIRED)
+  add_subdirectory(runtime-metrics/prometheus-reporter)
+  target_link_libraries(presto_server prometheus-cpp::core)
```

Comment:
This and `find_package(..)` above should be in `runtime-metrics/prometheus-reporter/CMakeLists.txt` as well.

Reply:
Done.

Reply:
We don't need them here after moving.